src/js/pages/services/ServicesTab.js

```diff
@@ -118,18 +119,33 @@ var ServicesTab = React.createClass({
         name: state.searchString
       }).getItems();
 
+      let breadcrumbsOrFilterHeadline = (
```

Comment:
This is a quantum variable. I don't know the content until I measure. :)
I am fine with it for now.
But it has some downsides like rafactorability/atomicity and separation of concerns.
I would like to see this in two separate variables next time. :)